### DISCUSSION TITLE
Having trouble connecting my BlueRetro to my pc

### DISCUSSION THREAD

**Discussion by Ultrapro-DC** [darthcloud/BlueRetro#731] [Q&A]
Hi there, I'd like to be able to change some of the mappings for my Dreamcast BlueRetro adapter and I am aware that I can connect it to Chrome via bluetooth, but I'm having trouble getting it to connect. I'd like to be able to map the 2nd thumbstick of my xbox series x controller to the mouse x/y axis for certain games such as Quake 3 Arena etc but I cannot get it to connect. I get the errors shown below when trying to connect. I use a USB Bluetooth dongle for my desktop but it does work and connects to bluetooth devices perfectly fine...

![image](https://github.com/darthcloud/BlueRetro/assets/139850936/75ccda56-0029-4517-83f1-68543913ee57)

![image-1](https://github.com/darthcloud/BlueRetro/assets/139850936/8bcb8671-8753-4c40-888b-a3e21c869330)

![image-2](https://github.com/darthcloud/BlueRetro/assets/139850936/18b15449-cca2-4f61-a67e-d315abddcc89)

**Comment by Rschillo19** [comment]
Same here - waiting on answers.  used numerous different devices.... pretty bummed so far with this product

**Reply by Gkjasl** [reply]
My experience was that the blue retro Bluetooth connection requires all nearby Bluetooth devices to have their Bluetooth disabled, except for the device you're trying to pair it with. I turned everything off but it still didn't pair! That's when I realized that my smartwatch on my wrist was Bluetooth enabled and jamming the signal.

Bluetooth soundbars and other devices can interfere as well.

**Comment by Rschillo19** [comment]
Took me 3 days to figure it it and working with laser bear but I was able
to get it to work by searching for it and getting the web app to lost the
game cube.  I'd highlight the gamecube but NOT CLICK PAIR yet.....

I'd turn the gamecube off and as soon as I turned the gamecube on I'd click
pair..... It took a few tries but it connected...... Very odd
